applications/FluidDynamicsApplication/custom_utilities/acceleration_limitation_utilities.cpp

```diff
@@ -0,0 +1,86 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+
+// Application includes
+#include "acceleration_limitation_utilities.h"
+
+namespace Kratos
+{
+
+    /* Public functions *******************************************************/
+
+    /**
+     * @brief Iteration over all nodes in ModelPart to find and reduce excessive accelerations.
+     * Model part is specified in the constructor
+     */
+
+    void AccelerationLimitationUtilities::Execute() {
+
+        double dt = mrModelPart.GetProcessInfo().GetValue(DELTA_TIME);
+
+        ModelPart::NodesContainerType rNodes = mrModelPart.Nodes();
+        #pragma omp parallel for
+        for(unsigned int count = 0; count < static_cast<unsigned int>(rNodes.size()); count++)
```

Comment:
this does not compile in windows, you have to use `int`

Reply:
Just to clarify what Philipp points, the problem in Windows is the OpenMP loop of an `unsigned int` variable. Of course you can loop `unsigned int` variables in Windows if you remove the OpenMP pragma.

Reply:
It was changed to "regular" int